tasks.py

```diff
@@ -84,6 +84,9 @@ def production_failsafe(ctx):
     It is meant to be prepended to the execution of other tasks to force a confirmation
     when a task is being executed that could have an impact on a production stack
     """
+    if not STACK_NAME:
+        raise Exit("No STACK_NAME specified")
+
```

Comment:
if prod identifier is an empty string and STACK_NAME is not defined, then the production failsafe will be triggered and then the user will get an error message that doesn't make it clear that the problem is that STACK_NAME is not defined